composer.json

```diff
@@ -2,119 +2,62 @@
     "name": "dpc-sdp/tide_core",
     "description": "Core functionality for Tide Drupal 8 distribution",
     "type": "drupal-module",
-    "license": "GPL-2.0+",
+    "license": "GPL-2.0-or-later",
     "require": {
-        "ckeditor/fakeobjects": "^4.5",
-        "ckeditor/iframe": "^4.5",
-        "drupal/address": "~1.0",
-        "drupal/allowed_formats": "^1.1",
-        "drupal/better_exposed_filters": "^3.0@alpha",
-        "drupal/ckeditor_iframe": "^2.0@beta",
-        "drupal/clamav": "^1.0",
-        "drupal/config_ignore": "^2.1",
-        "drupal/config_split": "^1.3",
-        "drupal/config_update": "^1.4",
-        "drupal/content_moderation_scheduled_updates": "^1.0",
+        "ckeditor/fakeobjects": "4.5.11",
+        "ckeditor/iframe": "4.5.11",
+        "drupal/address": "1.4",
+        "drupal/allowed_formats": "1.1",
+        "drupal/better_exposed_filters": "3.0-alpha5",
+        "drupal/ckeditor_iframe": "2.0-beta1",
+        "drupal/clamav": "1.1",
+        "drupal/config_ignore": "2.1",
+        "drupal/config_split": "1.4",
+        "drupal/config_update": "1.6",
+        "drupal/content_moderation_scheduled_updates": "1.0",
         "drupal/core": "8.5.6",
```

Comment:
this update to 8.6 is completely outside of this tasks. I have actually asked not to do this update until CI branches are merged, because it may require update of all other modules. Simply put - this is out of scope and budget. We will address this once it is all merged. Not sure what to do now with `master` being prematurely updated to `8.6`. Too many moving parts...